package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "dependencies": {
```

Comment:
This package.json is in the wrong place (should be `/frontend`.

And I actually don't think we need any of these packages; these are for nodejs backends I think.